tests/count_words_test.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+from src import count_words
+
+
+class TemplateTest(unittest.TestCase):
+
+    @staticmethod
+    def test_gets_words_from_string():
+        # Input data and reference
+        string_of_words_input = "Alon, guy , ~~: that are not words"
+        reference_words_list = ["Alon", "guy", "that", "are", "not", "words"]
+        # Check assertion
+        assert reference_words_list == count_words.gets_words_list_from_string(
+            string_of_words_input)
+
+    @staticmethod
+    def test_convert_to_lower_letter():
```

Comment:
name comment applies here as well. and this test also tests more than one thing - you tried to squeeze a different edge-case for each word in your list.